code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx

```diff
@@ -17,14 +19,19 @@ const OrderProductItem = ({
   quantity,
   price
 }: OrderProductItemProps) => {
+  const { price: productPrice, id, image, name, description } = product;
```

Comment:
if you give some renaming for price `price: productPrice`
maybe also give it for other properties line `name`, `image`, `description`
Or leave just `price` instead `price: productPrice` just everything in one approach

Reply:
Changed naming